src/foam/nanos/controller/ApplicationController.js

```diff
@@ -285,7 +288,28 @@ foam.CLASS({
     },
     'currentMenu',
     'lastMenuLaunched',
-    'webApp'
+    'webApp',
+    {
+      name   : 'XLIFFTranslation',
```

Comment:
Is this XLIFFTTranslation or LocaleTranslation? 
Is there something XLIFF speciific here?

Reply:
We should have a LocaleTranslation Interface that our XLIFFTransaction implements.